app/views/products/_show.html.haml

```diff
@@ -22,5 +22,5 @@
     %h3 Documentation
     %ul.unstyled
       - docs.each do |stored_file|
-        %li.document= link_to stored_file.name, stored_file.file.url
+        %li.document= link_to stored_file.name, stored_file.download_url
```

Comment:
We should probably put a bunch of these stored files behind a redirector. Otherwise we're going to run into the issue of users sitting on the page for a while and having the link expire.